src/main/scala/emm/effects/Binder.scala

```diff
@@ -29,6 +30,22 @@ trait BinderLowPriorityImplicits {
       NAP.pack(back)
     }
   }
+
+  implicit def binderState[S, C <: Effects](implicit C: Binder[C], T: Traverser[C], NN: NonNested[C]): Binder[State[S, ?] |: C] = new Binder[State[S, ?] |: C] {
```

Comment:
Question: do we want to support `State` and `StateT` separately?  `State` has some weird type inference issues, so it certainly wouldn't fit into any of the general cases.  The real question is how useful `State` is (vs `StateT` in a pivot).